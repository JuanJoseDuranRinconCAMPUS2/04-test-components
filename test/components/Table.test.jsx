import { render } from "@testing-library/react";
import Table from "../../src/components/Table"

describe("Captura del componente <Table.jsx>", ()=>{
    test('Fotito :3', () => { 
        let {container} = render(<Table/>);
        expect(container).toMatchSnapshot();
    })
})

//para cambiar el componenete y forzar a que se cambie la snapshot preciona "U" cuando haya error