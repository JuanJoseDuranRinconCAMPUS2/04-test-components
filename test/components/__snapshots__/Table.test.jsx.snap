// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente <Table.jsx> Fotito :3 1`] = `
<div>
  <div>
    Table
  </div>
</div>
`;
